### PR TITLE
fix nuget to allow c++/Winrt app to consume package

### PR DESCRIPTION
**Pull request by LPBourret**
This change correct the .targets include din .nuspec to correctly point to assets and pri files used to resolve runtime components and make them function as expected

### CODE REVIEW

File: samples/SentimentAnalyzerCustomSkill/build/FaceSentimentAnalyzer_CS.targets [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <FaceSentimentAnalyzer-Platform Condition="'$(Platform)' == 'Win32'">x86</FaceSentimentAnalyzer-Platform>
+        <FaceSentimentAnalyzer-Platform Condition="'$(Platform)' != 'Win32'">$(Platform)</FaceSentimentAnalyzer-Platform>
+    </PropertyGroup>
+    <ItemGroup>
+        <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0.17763\Contoso.FaceSentimentAnalyzer.winmd" />
+        <!--<ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\contentFiles\any\uap10.0.17763\Contoso.FaceSentimentAnalyzer.pri" /> -->
+    </ItemGroup>
+</Project>
```
Comment (LPBourret) [inline samples/SentimentAnalyzerCustomSkill/build/FaceSentimentAnalyzer_CS.targets]: will remove this file

File: samples/SentimentAnalyzerCustomSkill/build/Contoso.FaceSentimentAnalyzer_CPP.targets [hunk 2]
```diff
@@ -9,6 +9,12 @@
             <Implementation>Contoso.FaceSentimentAnalyzer.dll</Implementation>
         </Reference>
         <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(FaceSentimentAnalyzer-Platform)\native\Contoso.FaceSentimentAnalyzer.dll" />
-		<ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\contentFiles\any\Contoso.FaceSentimentAnalyzer.dll.pri" />
+		<ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\contentFiles\any\uap10.0.17763\Contoso.FaceSentimentAnalyzer.pri" />
+		
+		<Reference Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0.17763\DeobfuscationHelper.winmd">
+            <Implementation>DeobfuscationHelper.dll</Implementation>
+        </Reference>
+		<ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(FaceSentimentAnalyzer-Platform)\native\DeobfuscationHelper.dll" />
```
Comment (dhung-msft) [inline samples/SentimentAnalyzerCustomSkill/build/Contoso.FaceSentimentAnalyzer_CPP.targets]: nit: tabs/spaces

File: samples/SentimentAnalyzerCustomSkill/build/Contoso.FaceSentimentAnalyzer_CS.targets [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <FaceSentimentAnalyzer-Platform Condition="'$(Platform)' == 'Win32'">x86</FaceSentimentAnalyzer-Platform>
+        <FaceSentimentAnalyzer-Platform Condition="'$(Platform)' != 'Win32'">$(Platform)</FaceSentimentAnalyzer-Platform>
+    </PropertyGroup>
+    <ItemGroup>
+        <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0.17763\Contoso.FaceSentimentAnalyzer.winmd" />
+		<ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0.17763\Contoso.FaceSentimentAnalyzer.pri" />
```
Comment (dhung-msft) [inline samples/SentimentAnalyzerCustomSkill/build/Contoso.FaceSentimentAnalyzer_CS.targets]: nit: inconsistent spacing

File: samples/SentimentAnalyzerCustomSkill/build/Contoso.FaceSentimentAnalyzer_CPP.targets (hunk 2 above)
Comment (dhung-msft) [reply samples/SentimentAnalyzerCustomSkill/build/Contoso.FaceSentimentAnalyzer_CPP.targets]: same throughout the PR

**Pull request merged**